src/functions/template-tags/organizer.php

```diff
@@ -243,6 +243,11 @@ function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = fa
 				return apply_filters( 'tribe_get_organizer_link', $link, $postId, $full_link, $url );
 			}
 		}
+
+		if ( ! $full_link ) {
```

Comment:
I'm _slightly_ concerned by this change.

* It doesn't take account of `$echo` (though really that's a structural problem with this function)
* If `$echo` *and* `$full_link` are both false the result will always be the same - an empty string - even when it could return the organizer name, but unlinked

Looking over the [linked issue](https://central.tri.be/issues/69821) to get a better handle on things, the experience reported there is also at odds with what I experience locally (develop branch). Could this be worth a second look?

Imho we should take this opportunity to also fix the phpdoc for this function, the notes for the `$full_link` param are misleading (it returns the organizer name, **not** the URL, when set to false - or at least that was the original intent before this change).